tests/check_consistent.py

```diff
@@ -151,6 +151,12 @@ def check_metadata() -> None:
             for key in data.get("tool", {}).get(tool, {}):
                 assert key in tk, f"Unrecognised {tool} key {key} for {distribution}"
 
+        stubtest_platforms = set(data.get("tool", {}).get("stubtest", {}).get("platforms", []))
+        supported_platforms = {"win32", "linux", "darwin"}
```

Comment:
I'd consider moving the supported platforms to the top of the file.

Reply:
Yes, that makes sense. At some point I'd like to centralise that information between tests as well (but not in this PR).